master/getting-started/kubernetes/upgrade/migrate.md

```diff
@@ -1,35 +1,52 @@
 ---
-title: Migrating data from etcdv2 to etcdv3
+title: Migrating Calico data
 no_canonical: true
 ---
 
 > **Important**: Once you begin the migration, stop using `calicoctl` or 
-> otherwise modifying the etcdv2 datastore. Such changes are unlikely to 
-> be migrated to the new datastore.
+> otherwise modifying the etcdv2 datastore. Any changes to etcdv2
+> data are unlikely to be migrated to the new datastore.
 {: .alert .alert-danger}
 
-1. Run `calico-upgrade start` to start the v1 to v3 data migration. This begins 
-   an interactive session. While existing connectivity will continue as before,
-   you cannot add any new endpoints until the migration and upgrade complete.
+
+1. To begin an interactive data migration session, use the `start` command. While 
+   existing connectivity will continue as before, you cannot add any new endpoints 
+   until the migration and upgrade complete.
+
+   **Syntax**
+   ```
+   calico-upgrade[-darwin-amd64|-windows-amd64.exe] start [--apiconfigv1 path/file] [--apiconfigv3 path/file]
```

Comment:
I think adding in the darwin/windows suffixes here is unnecessary and makes it harder to parse. WDYT about removing it?

Reply:
This is the syntax part of it so it should include everything I think.... But we could start with the example since it is the simplest case.

Reply:
Personally, I ended up running the Linux build accidentally because it's easy to miss that there are the different builds.